Website: Add meta descriptions to Fleet documentation. (#12586)
fleetdm/fleet: docs/Using-Fleet/Troubleshooting-live-queries.md

If this works and the browser is not working then it might be a rendering issue on the browser.
You should also try running the live query on different browsers.

<meta name="pageOrderInSection" value="1800">

> <meta name="description" value="This page contains an overview and troubleshooting steps for live queries in Fleet.">

Reviewer: ```suggestion
<meta name="description" value="An overview of live queries in Fleet and steps for troubleshooting.">
```
